packages/rest-hooks/tests/restClient.test.ts

```diff
@@ -128,502 +124,552 @@ describe('Test Rest Client', () => {
+      } 
+    }
+    assertResponse(rsp, expectedRsp)
 
-  //   const getOneFnWithPathParams = restClient.createGetFn('/simple-get/:id', {
-  //     pathParams: { id: 33 }
-  //   })
+    // with path params and query params
 
-  //   rsp = await getOneFnWithPathParams() as unknown as StcRestTest.TestResponse
+    const getOneFnWithQueryAndPathParams = restClient.createGetFn('/simple-get/:enityType', {
+      pathParams: { enityType: 'users' },
+      queryParams: { limit: 100, offset: 0 }
+    })
 
-  //   expectedRsp = [{
-  //     url: 'http://fakehost2.com:7777/simple-get/33',
-  //     method: 'GET',
-  //     headers: { authorization: 'Bearer testing-access-token' },
-  //   }, { 'simpleGet': 'data' }]
+    rsp = await getOneFnWithQueryAndPathParams() as unknown as StcRestTest.TestResponse
 
-  //   assertResponse(rsp, expectedRsp)
+    expectedRsp = {
+      data: { 'simpleGet': 'data' },
+      meta: {
+        url: 'http://fakehost2.com:7777/simple-get/users?limit=100&offset=0',
+        method: 'GET',
+        headers: { authorization: 'Bearer testing-access-token' },
+      } 
+    }
+    assertResponse(rsp, expectedRsp)
+  })
 
-  //   // with query params
+  test('Test Straight Post', async () => {
+    const restClient = createRestClient(defaultClientConfig, defaultServerConfig)
+    const postData = { name: 'bill' }
 
-  //   const getManyFnWithQueryParams = restClient.createGetFn('/simple-get', {
-  //     queryParams: { hydrate: true, paginate: false }
-  //   })
+    rsp = await restClient.post(
+      '/simple-post', postData) as unknown as StcRestTest.TestResponse
 
-  //   rsp = await getManyFnWithQueryParams() as unknown as StcRestTest.TestResponse
+    expectedRsp = {
+      data: {
+        message: 'post successful'
+      },
+      meta: {
+        url: 'http://fakehost.com:5023/simple-post',
+        method: 'POST',
+        headers: { authorization: 'Bearer testing-access-token' },
+        body: postData,
+      } 
+    }
+    assertResponse(rsp, expectedRsp)
+  })
 
-  //   expectedRsp = [{
-  //     url: 'http://fakehost2.com:7777/simple-get?hydrate=true&paginate=false',
-  //     method: 'GET',
-  //     headers: { authorization: 'Bearer testing-access-token' },
-  //   }, { 'simpleGet': 'data' }]
+  test('Test PostFn', async () => {
 
-  //   assertResponse(rsp, expectedRsp)
+    const restClient = createRestClient({
+      ...defaultClientConfig,
+      getAccessToken: () => 'another-access-token'
+    }, defaultServerConfig)
 
-  //   // with path params and query params
+    // simple post
 
-  //   const getOneFnWithQueryAndPathParams = restClient.createGetFn('/simple-get/:enityType', {
-  //     pathParams: { enityType: 'users' },
-  //     queryParams: { limit: 100, offset: 0 }
-  //   })
+    const postFn = restClient.createPostFn('/simple-post')
+    body = { species: 'dog', breed: 'lab' }
+    rsp = await postFn({ data: body }) as unknown as StcRestTest.TestResponse
```

Comment:
Now that we pass body to the postFn function, should update the `MutateFnOptions` type to be `bodyData` instead? 

`  /**
    Options passed into the {@link MutateFn}.
  */
  export interface MutateFnOptions {
    data: unknown;`// to bodyData?
      // The parameters for the mutation operation.
    restParams?: RestParams;
      // restParams.pathParams - For path variable substitutions.
      // restParams.queryParams - Optional path and query params for the REST call.
  }`